tests/inspector/ckeditorinspector.js

```diff
@@ -70,31 +95,165 @@ describe( 'CKEditorInspector', () => {
 				} );
 		} );
 
-		it( 'attaches a new editor (named)', () => {
-			CKEditorInspector.attach( 'foo', editor );
+		it( 'attaches to a named editor', () => {
+			CKEditorInspector.attach( { foo: editor } );
 
 			inspectorRef = CKEditorInspector._inspectorRef.current;
 
 			expect( inspectorRef.state.editors.get( 'foo' ) ).to.equal( editor );
 		} );
 
+		it( 'attaches to multiple editors at a time', () => {
+			return TestEditor.create( element )
+				.then( anotherEditor => {
+					const names = CKEditorInspector.attach( { foo: editor, bar: anotherEditor } );
+
+					inspectorRef = CKEditorInspector._inspectorRef.current;
+
+					expect( inspectorRef.state.editors.get( 'foo' ) ).to.equal( editor );
+					expect( inspectorRef.state.editors.get( 'bar' ) ).to.equal( anotherEditor );
+					expect( names ).to.have.members( [ 'foo', 'bar' ] );
+
+					return anotherEditor.destroy();
+				} )
+				.catch( err => {
+					throw err;
+				} );
+		} );
+
+		it( 'supports the deprecated syntax but warns', () => {
+			CKEditorInspector.attach( 'foo', editor );
+
+			inspectorRef = CKEditorInspector._inspectorRef.current;
+			expect( inspectorRef.state.editors.get( 'foo' ) ).to.equal( editor );
+			sinon.assert.calledOnce( warnStub );
+			sinon.assert.calledWithMatch( warnStub, /^\[CKEditorInspector\]/ );
+		} );
+
 		it( 'detaches when the editor is destroyed', () => {
 			const spy = sinon.spy( CKEditorInspector, 'detach' );
 
-			CKEditorInspector.attach( 'bar', editor );
+			CKEditorInspector.attach( { bar: editor } );
 
 			return editor.destroy().then( () => {
 				sinon.assert.calledOnce( spy );
 				sinon.assert.calledWith( spy, 'bar' );
 
+				inspectorRef = CKEditorInspector._inspectorRef.current;
+
 				expect( inspectorRef.state.editors.size ).to.equal( 0 );
 			} );
 		} );
+
+		describe( 'options', () => {
+			describe( '#isCollapsed', () => {
+				beforeEach( () => {
+					CKEditorInspector.destroy();
```

Comment:
It should not be needed - https://github.com/ckeditor/ckeditor5-inspector/pull/58/files#diff-d1ba5b75e77f433734c13ba3d41e720cR34